class/airlock-microgateway.yml

```
@@ -1,11 +1,94 @@
 parameters:
-  kapitan:
-    compile:
-      - input_paths:
-          - ${_base_directory}/component/app.jsonnet
-        input_type: jsonnet
-        output_path: .
-      - input_paths:
-          - ${_base_directory}/component/main.jsonnet
+  =_helm_chart_name: "microgateway"
+  =_release_name: "airlock-microgateway"
+  =_kapitan:
+    k8s_gw_api:
+      'True':
+        input_paths:
+          - ${_base_directory}/dependencies/gateway-api/standard-install.yaml
+        input_type: copy
+        output_path: airlock-microgateway/00_prerequisites/00_gateway_api/
+      'False':
+        input_paths: []
         input_type: jsonnet
-        output_path: airlock-microgateway/
+        output_path: ''
+    olm:
+      dependencies:
+        - type: https
+          source: ${airlock_microgateway:gateway_api:source}
+          output_path: ${_base_directory}/dependencies/gateway-api/standard-install.yaml
```

why not just put this in `manifests/` which is intended for downloaded K8s manifests?

```suggestion
          output_path: ${_base_directory}/manifests/gateway-api/standard-install.yaml
```